modules/docs/arrow-docs/docs/docs/patterns/typeclasspolymorphism/README.md

```diff
@@ -0,0 +1,906 @@
+---
+layout: docs
+title: Monad Comprehensions
+permalink: /docs/patterns/typeclass_polymorphism/
+---
+
+## TypeClass Polymorphism
+
+{:.advanced}
+advanced
+
+What if we could write apps without caring about the runtime data types used but just about __how the data is operated 
+on__? 
+
+Let's say we have an  application working with RxJava's `Observable`. We'll end up having a bunch of chained call stacks 
```

Comment:
Please link back to  `({{ '/docs/integrations/rx2' | relative_url }})`. I'l flag a few of these more, just with the address instead.

Reply:
I'm actually talking about standard containers here, not the HKs. Does it make sense to link back to the wrappers?

Reply:
You're right in this case, yes.